test/codeblock.html

```diff
@@ -1,4 +1,4 @@
 <p>Here is a square:</p>
 <div class="sourceCode" id="cb1" data-echo="Above"><pre
 class="sourceCode haskell"><code class="sourceCode haskell"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a>example <span class="ot">=</span> square <span class="dv">1</span></span></code></pre></div>
-<p><img src="image.png" /></p>
```

Comment:
Apparently I have introduced a regression in b6582a840ac17acdafd0bc52b38ec32727656241, where I unnecessarily wrapped what should have been inline images with paragraphs. This PR fixes that as well.